src/tally.F90

```diff
@@ -1230,20 +1247,21 @@ subroutine score_general_mg(p, t, start_index, filter_index, i_nuclide, &
           else
             score = p % last_wgt
           end if
+
           if (i_nuclide > 0) then
-            score = score * nucxs % get_xs('inv_vel', p_g, UVW=p_uvw) / &
-                 matxs % get_xs('total', p_g, UVW=p_uvw) * flux
+            score = score * nucxs % get_xs('inverse-velocity', p_g, UVW=p_uvw) &
+                 / nucxs % get_xs('total', p_g, UVW=p_uvw) * flux
```

Comment:
Not sure. I changed it back. Also I changed `total` for `absorption` to be consistent with the other scores.